src/control/control.c

```diff
@@ -667,9 +668,19 @@ int dt_control_key_pressed_override(guint key, guint state)
   if(darktable.view_manager->current_view->dynamic_accel_current)
   {
     gchar **vals = g_strsplit_set(darktable.view_manager->current_view->dynamic_accel_current->translated_path, "/", -1);
+    dt_iop_module_so_t *mod_so = darktable.view_manager->current_view->dynamic_accel_current->mod_so;
+    dt_iop_module_t *mod = dt_iop_get_module_accel_curr(mod_so);
     if(vals[0] && vals[1] && vals[2] && vals[3])
     {
-      gchar *txt = dt_util_dstrcat(NULL, _("scroll to change <b>%s</b> of %s module"), vals[3], vals[2]);
+      gchar *txt;
+      if(mod->multi_name && mod->multi_name[0] != '\0')
```

Comment:
This generates following error:
```
/build/darktable/src/control/control.c:676:15: error: address of array 'mod->multi_name' will always evaluate to 'true' [-Werror,-Wpointer-bool-conversion]

      if(mod->multi_name && mod->multi_name[0] != '\0')

         ~~~~~^~~~~~~~~~ ~~
```

Reply:
There wasn't any real need for the if statement any more so I removed it in latest commit. You'll get an extra space in the hint if there's only one version of the module but I don't think that's a big deal.